.pre-commit-config.yaml

```diff
@@ -1,19 +1,20 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
+    rev: v6.0.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-docstring-first
 -   repo: https://github.com/PyCQA/flake8
-    rev: 3.9.2
+    rev: 7.3.0
     hooks:
     -   id: flake8
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v1.5.7
+        args: ["--config=.flake8"]
```

Comment:
This is also not needed

Reply:
I tried to suppress one of the flake8 rules because it gave 100% wrong erros on strings.